<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hardWindow</class>
 <widget class="QMainWindow" name="hardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1397</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>8</y>
      <width>1391</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="hard.qrc">:/hard/hard1- 121.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>0</y>
      <width>1391</width>
      <height>701</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="hard.qrc">:/hard/hard1- 122.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1391</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="hard.qrc">:/hard/hard1- 123.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>8</y>
      <width>1391</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="hard.qrc">:/hard/hard1- 124.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>9</y>
      <width>1391</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="hard.qrc">:/hard/hard1- 125.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>10</y>
      <width>1391</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="hard.qrc">:/hard/hard1- 126.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>8</y>
      <width>1391</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="hard.qrc">:/hard/hard1- 127.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>10</y>
      <width>1391</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="hard.qrc">:/hard/hard1- 128.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>-110</x>
      <y>-10</y>
      <width>1391</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="hard.qrc">:/hard/hard1- 129.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="hclock">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>20</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>70</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>return</string>
    </property>
   </widget>
   <widget class="QLabel" name="hdrum1">
    <property name="geometry">
     <rect>
      <x>254</x>
      <y>108</y>
      <width>171</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="drum.qrc">:/new/prefix1/drum- 7.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="hdrum2">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>118</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="drum_2.qrc">:/drum_2/drum2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="hdrum3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>98</y>
      <width>171</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="drum.qrc">:/new/prefix1/drum- 7.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="hdrum4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>88</y>
      <width>171</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="drum_2.qrc">:/drum_2/drum2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="hdrum5">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>70</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="drum.qrc">:/new/prefix1/drum- 7.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="hdrum6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>48</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="drum_2.qrc">:/drum_2/drum2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="flower">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>598</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="fl.qrc">:/flo/f - 13.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="hscorenum">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>-2</y>
      <width>1391</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="re.qrc">:/all/scorewindow.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="score_score">
    <property name="geometry">
     <rect>
      <x>594</x>
      <y>148</y>
      <width>271</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>600</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>restart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>594</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1397</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="drum.qrc"/>
  <include location="re.qrc"/>
  <include location="drum_2.qrc"/>
  <include location="hard.qrc"/>
  <include location="fl.qrc"/>
 </resources>
 <connections/>
</ui>
